package.json

```diff
@@ -490,18 +490,19 @@
                         "string",
                         "array"
                     ],
-                    "default": "",
+                    "default": [],
                     "description": "Defines which files in the workspace to ignore for Zip deploy. This applies to Zip deploy only, has no effect on other deployment methods."
                 },
                 "appService.showRemoteFiles": {
                     "type": "boolean",
                     "default": false,
                     "description": "Show or hide the File Nodes of a Web App.  Caution: Manually editing your Web App's files could cause unexpected behavior."
                 },
-                "appService.showSavePrompt": {
+                "appService.neverPromptBuildDuringDeploy": {
+                    "scope": "resource",
```

Comment:
This flag doesn't match the other flags. Do something like "showBuildDuringDeployPrompt"